extensions/reactor-client/src/test/java/com/aerospike/client/reactor/ReactorIndexTest.java

```diff
@@ -0,0 +1,47 @@
+package com.aerospike.client.reactor;
+
+import com.aerospike.client.query.IndexCollectionType;
+import com.aerospike.client.query.IndexType;
+import com.aerospike.client.reactor.util.Args;
+import org.junit.Test;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+public class ReactorIndexTest extends ReactorTest{
+
+    public ReactorIndexTest(Args args) {
+        super(args);
+    }
+
+    @Test
+    public void shouldCreateAndDropIndex() {
+
+        String indexName = "test_index";
+
+        Mono<Boolean> indexExists = reactorClient.indexExists(null, args.namespace, indexName);
+        StepVerifier.create(indexExists)
+                .expectNext(false)
+                .verifyComplete();
```

Comment:
If we call create index with the same params what is the expected result? Could you please cover that with the test? Probably the same for delete operation would be nice to have as well

Reply:
covered both cases with specific tests